### DISCUSSION TITLE
加入频道后必须重启bot才能加载出来，可不可以实时加载

### DISCUSSION THREAD

**Discussion by 250king** [Mrs4s/go-cqhttp#1275] [Q&A]
bot已进入一个频道，但弄半天发不了消息，折腾半天才发现没加载出来，重启了bot就恢复正常了，但每次加入一个频道的时候还要重启一次bot感觉好麻烦，可不可以像Mirai一样实时加载

**Comment by Mrs4s** [comment]
还在适配中
